cron/daily/92-refresh-expense-tag-stats-materialized-view.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
```

Comment:
I think you should include this in `cron/hourly/50-refresh-materialized-views.js`.

Reply:
Ah, nice, had not seen that one. Included it there, also moved the daily cron of refreshing the CollectiveTagStats view to this file.